prism/static/js/src/studio.js

```diff
@@ -0,0 +1,33 @@
+/* Javascript for PrismXBlock. */
+function PrismXBlockEditBlock(runtime, element) {
+    var $element = $(element);
+
+    var lngSelect = document.getElementById('language');
+    var thmSelect = document.getElementById('theme');
+
+    var codeMirror = CodeMirror.fromTextArea(document.querySelectorAll('textarea#code-textarea')[0], {
+        lineNumbers: true,
+        indentWithTabs: true,
+        matchBrackets: true,
+        lineWrapping: true,
+    });
+
+    $(element).find('.save-button').bind('click', function(){
+        var handlerUrl = runtime.handlerUrl(element, 'studio_submit'); 
+        var data = {
+            display_name: $(element).find('input[name=display_name]').val(),
+            code_data: $(element).find('#code-textarea').val(),
+            language: lngSelect.options[lngSelect.selectedIndex].text,
+            theme: thmSelect.options[thmSelect.selectedIndex].text,
+            height: $(element).find('input[name=height]').val(),
+        };
+        // runtime.notify('save', {state: 'start'});
```

Comment:
**nit:** I don't exactly know what this line do, but it seems useful 😅 

```suggestion
        runtime.notify('save', {state: 'start'});
```

Reply:
@OmarIthawi the `runtime.notify` are commented out for now because they are throwing errors the dev env and disabling the `post` method. I will uncomment when I test in devstack.